servicebus/data-plane/azure-servicebus/pom.xml

```diff
@@ -1,17 +1,35 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Can you go through this and convert indents to spaces.

Reply:
It is easier to view.